src/pages/app/exams/details/[examId].tsx

```
@@ -66,80 +77,345 @@ const ExamDetails = () => {
     status = "Active";
   }
 
-  const leaderboard = data.leaderboard && data.leaderboard.length > 0 ? data.leaderboard : []; // mockLeaderboard;
-  const participants = data.participants ?? []; // mockParticipants;
+  const leaderboard = data.leaderboard && data.leaderboard.length > 0 ? data.leaderboard : [];
+  const participants = data.participants ?? [];
+
+  const { showToast } = useToast();
```

ESLint

[error] React Hook "useToast" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)